src/shared/chat/emoji-dropdown.js

```diff
@@ -95,6 +95,12 @@ export default class EmojiDropdown extends DropdownBase {
         super.showMenu();
         setTimeout(() => this.querySelector('.emoji-search')?.focus());
     }
+
+    hideMenu () {
+        this.chatview.querySelector('converse-emoji-picker')?.disableArrowNavigation();
```

Comment:
In `src/shared/chat/emoji-picker.js` there is a function `_onGlobalKeyDown`. I think it would be better to listen for the escape key and then to call `disableArrowNavigation` there, instead of doing it here.